docs/source/algorithms/CalculateSelfScatteringCorrection-v1.rst

```diff
@@ -0,0 +1,31 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This is a workflow algorithm that calculates the placzek self scattering
+factor focused by into detector banks. This is done by executing several
```

Comment:
*focused into detector banks*?

Reply:
the output from the CalculatePlaczekSelfScattering algorithm is given as a value for each pixel, that is then foccused by this algorithm into the indevidual banks and then normalized by number of pixel per bank.